### ISSUE TITLE
Strange behavior with Sympy

### ISSUE THREAD

**Issue by JaredCBronski** [PrairieLearn/PrairieLearn#4793]
I have found some strange behavior when coding some symbolic problems in PL. The server.py code is 

```
import sympy
import prairielearn as pl

def generate(data):
    sympy.var('V R r C t t1')
    T = sympy.ln(2)*(r+2*R)*C 
    data['correct_answers']['T'] = pl.to_json(T)

```

and the question.html file is 

`<pl-symbolic-input answers-name="T" variables="r, R, C, V" label="Period $~~T=$"></pl-symbolic-input>`

It will throw errors on certain inputs but the pattern is not clear. For instance the input `log(-V)/-(R+r)*C` throws an error but the algebraically equivalent` log(-V)/-C*(R+r)` does not.

**Comment by jonatanschroeder** [comment]
This seems to be a problem with sympy itself.

```
Python 3.8.5 (default, May 27 2021, 13:30:53)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sympy.parsing.sympy_parser import parse_expr
>>> x = parse_expr("log((2*V/3-V)/C)/-(R+r)*C")
>>> x.equals(x)
True
>>> y = parse_expr("log((2*V/3-V)/C)/-(R+r)*2")
>>> x.equals(y)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/core/expr.py", line 751, in equals
    constant = diff.is_constant(simplify=False, failing_number=True)
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/core/expr.py", line 674, in is_constant
    if b is not None and b is not S.NaN and b.equals(a) is False:
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/core/expr.py", line 741, in equals
    diff = factor_terms(simplify(self - other), radical=True)
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/simplify/simplify.py", line 635, in simplify
    _e = cancel(expr)
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/polys/polytools.py", line 6688, in cancel
    f = factor_terms(f, radical=True)
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/core/exprtools.py", line 1274, in factor_terms
    return do(expr)
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/core/exprtools.py", line 1271, in do
    rv = _keep_coeff(cont, p, clear=clear, sign=sign)
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/core/mul.py", line 2069, in _keep_coeff
    if r == int(r):
  File "/home/jonatan/.local/lib/python3.8/site-packages/sympy/core/expr.py", line 332, in __int__
    raise TypeError("can't convert %s to int" % r)
TypeError: can't convert oo to int
>>>
```

**Comment by echuber2** [comment]
Looks like this has been fixed in Sympy but the update may not have filtered down through pip yet.

**Comment by eliotwrobson** [comment]
Has this been fixed? This issue has been open for a while, and seems like it would have reached PL by now.

**Comment by nwalters512** [comment]
Per https://github.com/sympy/sympy/pull/22024, this was released as part of Sympy 1.9, and we're currently running 1.11.1. I tried Jonatan's example and it now completes without an error:

```python
>>> from sympy.parsing.sympy_parser import parse_expr
>>> x = parse_expr("log((2*V/3-V)/C)/-(R+r)*C")
>>> x.equals(x)
True
>>> y = parse_expr("log((2*V/3-V)/C)/-(R+r)*2")
>>> x.equals(y)
False
```
